akka-persistence-query/src/main/scala/akka/persistence/query/typed/EventEnvelope.scala

```diff
@@ -94,7 +133,32 @@ final class EventEnvelope[Event](
     val entityType: String,
     val slice: Int,
     val filtered: Boolean,
-    val source: String) {
+    val source: String,
+    val tags: Set[String]) {
```

Comment:
do we have a serializer that needs to be updated?

Reply:
Didn't think there was anything else than the gRPC transport but we do have serialisation for it in QuerySerializer, I'll update

Reply:
I think it's sent over pubsub